framework/src/modules/chain/dpos/delegates_info.js

```diff
@@ -81,23 +81,18 @@ class DelegatesInfo {
 		 * the genesis block, in that case we don't have to
 		 * update anything in the accounts.
 		 */
-		if (block.height === 1) {
+		if (this._isGenesisBlock(block)) {
+			const round = 1;
+			await this.delegatesList.createRoundDelegateList(round, tx);
```

Comment:
@shuse2 @lsilvs do we need to create a delegate list for the first round?

Reply:
<img width="1343" alt="Screenshot 2019-09-13 at 21 49 45" src="https://user-images.githubusercontent.com/26482710/64890653-75a5eb80-d670-11e9-9de9-7e80b8981dc3.png">

We decided to add this as otherwise we got the following error.

Reply:
Just for the record, we do need to get the delegate list for the first round, and we were doing it before too 😉